File: exercicios/R/1181/1181.r
input = file('stdin','r')

soma = as.double(0)
neg = as.double(0)
matriz = matrix(c(144),nrow= 12, ncol = 12, byrow = TRUE)
linha = as.integer(readLines(input,n=1))

op = readLines(input,n=1)



for(i in 1:144){
    valor = as.double(readLines(input,n=1))
    matriz[i] = valor
}

for(j in 1:12){
        soma = soma + matriz[j,linha]
}

if(op == 'S'){
    write(sprintf("neg %.1f",neg),'')
    write(sprintf("soma %.1f",soma),'')
}else{
    media = as.double(soma/12)
    write(sprintf("%.1f",media),'')
}
print(matriz)
for(j in 1:12){
        write(matriz[j,linha],'')
}

File: exercicios/JavaScript/1181/1181.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n')

var matriz = Array(12).fill(null).map(() => Array(12).fill(0));
var soma = 0

var linha = parseInt(lines.shift())

var op = String(lines.shift())

for(let i=0; i<12; i++){
    for(let j=0; j<12; j++){
        var v = parseFloat(lines.shift())
        matriz[i][j] = v
    }
}

for(let k=0; k<12; k++){
    soma += matriz[linha][k]
    //console.log(matriz[linha][k])
}
if(op == 'S\r' || op == 'S'){
    console.log(soma.toFixed(2))
}else if(op == 'M\r' || op == 'M'){
    var media = soma / 12
    console.log(media.toFixed(1))
}

//console.table(matriz)